tests/frontend/github_repo_info_controller_test.js

```diff
@@ -69,6 +69,32 @@ describe("GitHub Repo Info controller", () => {
     });
   });
 
+  it("not-found response hides", (done) => {
+    fetch.mockResponse(
+      JSON.stringify({
+        html_url: "https://github.com/ghost/none",
+        stargazers_count: 100,
+        forks_count: 200,
+        open_issues_count: 300,
```

Comment:
Note: this is mostly copied from the mock valid API response in the same file.  Perhaps there'd be alternative, better test fixture data to return from here.

Reply:
The current 404 response looks like this:
```json
{
  "message": "Not Found",
  "documentation_url": "https://docs.github.com/rest/reference/repos#get-a-repository"
}